src/test/java/Homework17.java

```diff
@@ -0,0 +1,53 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest{
+    @Test
+    public void addSongToPlaylist(){
+
+        EdgeOptions options = new EdgeOptions();
+        options.addArguments("--remote-allow-origins=*");
+
+        WebDriver driver = new EdgeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String baseUrl = "https://qa.koel.app/";
+        driver.get(baseUrl);
+
+        WebElement emailField = driver.findElement(By.cssSelector("input[type='email']"));
+        emailField.clear();
+        emailField.sendKeys("john.wolschleger@testpro.io");
+
+        WebElement passwordField = driver.findElement(By.cssSelector("input[type='password']"));
+        passwordField.clear();
+        passwordField.sendKeys("te$t$tudentb49");
+
+        WebElement submit = driver.findElement(By.cssSelector("button[type='submit']"));
+        submit.click();
+
+        WebElement avatarIcon = driver.findElement(By.cssSelector("img[class='avatar']"));
+        Assert.assertTrue(avatarIcon.isDisplayed());
+
+        WebElement searchField = driver.findElement(By.cssSelector("input[type='search']"));
+        searchField.clear();
+        searchField.sendKeys("Pluto");
+
+        WebElement viewAll = driver.findElement(By.cssSelector("button[data-test='view-all-songs-btn']"));
+        viewAll.click();
+
+        WebElement addTo = driver.findElement(By.cssSelector("button[class='btn-add-to']"));
+        addTo.click();
+
+        WebElement verifySongToPlaylist = driver.findElement(By.cssSelector("div.alertify-logs.top.right"));
+        Assert.assertTrue(verifySongToPlaylist.isDisplayed());
+
+        driver.quit();
+    }
```

Comment:
If possible, avoid overly long methods. A long method is 10‒15 lines. If possible, you should split the method into several shorter ones 

   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**